tutorial/libFuzzerTutorial.md

```diff
@@ -426,7 +426,7 @@ Try this with one of the crashes you have found previously.
 We recommend [Clang Coverage](http://clang.llvm.org/docs/SourceBasedCodeCoverage.html) to visualize and study your code coverage. A simple example:
 ```
 # Build you code for Clang Coverage; link it against a standalone driver for running fuzz targets.
-svn co http://llvm.org/svn/llvm-project/compiler-rt/trunk/lib/fuzzer Fuzzer
+svn export http://github.com/llvm/llvm-project/trunk/compiler-rt/lib/fuzzer Fuzzer
```

Comment:
just switch to 
git clone http://github.com/llvm/llvm-project
cd lib/fuzzer

Reply:
Thanks for your reply.

Installing libfuzzer's source codes through "git clone" will produce a much bigger installation volume (i.e., we will install the whole llvm project which contains way more irrelevant codes and a ".git" folder). 

Here are some statistics:

![image](https://user-images.githubusercontent.com/38074777/117157248-048cec00-adf1-11eb-9016-cbd898abe40e.png)

Reply:
You can try sparseCheckout and pull with --depth=1

```
mkdir llvm-project
cd llvm-project
git init
echo "compiler-rt/" >> .git/info/sparse-checkout
git config core.sparseCheckout true
git remote add origin http://github.com/llvm/llvm-project 
git pull --depth=1 origin 
git checkout origin/main
```

Reply:
@vitalybuka Thank you for the reply and this is definitely an awesome workaround as well.

I think the `svn export` method is mostly compatible with the previous `svn co` command:
1. it is simple (1 line);
2. it fetches the `fuzzer` folder only without installing other stuff;
3. it pops the content in the same place (previous `svn co` also pops the in-coming folder as `./Fuzzer`)

A possible shortcoming for `svn` that I could imagine is that on macOS not everybody has an svn (they may need to `brew install svn`).

Reply:
@inferno-chromium Hi, any follow-ups?

Reply:
Sure lets keep as it is for simplificity.